packages/fxa-settings/src/components/ConnectedServices/index.test.tsx

```diff
@@ -4,17 +4,128 @@
+    });
+
+    const sortedList = sortAndFilterConnectedClients(MOCK_SERVICES);
+
+    expect(sortedList.length).toEqual(7);
+
+    expect(
+      sortedList.filter((item) => item.name === 'Firefox Monitor').length
+    ).toEqual(1);
+  });
+
+  it('should show the pocket icon and link', async () => {
+    const servicesList = MOCK_SERVICES.filter((item) => item.name === 'Pocket');
+    renderWithRouter(
+      <MockedCache account={{ attachedClients: servicesList }}>
+        <ConnectedServices />
+      </MockedCache>
+    );
+    expect(await screen.findByTestId('pocket-icon')).toBeTruthy;
+    expect(screen.getByTestId('service-name')).toHaveAttribute(
+      'href',
+      'https://www.mozilla.org/en-US/firefox/pocket/'
+    );
+  });
+
+  it('should show the monitor icon and link', async () => {
+    const servicesList = MOCK_SERVICES.filter(
+      (item) => item.name === 'Firefox Monitor'
+    );
+    renderWithRouter(
+      <MockedCache account={{ attachedClients: servicesList }}>
+        <ConnectedServices />
+      </MockedCache>
+    );
+    expect(await screen.findByTestId('monitor-icon')).toBeTruthy;
+    expect(screen.getByTestId('service-name')).toHaveAttribute(
+      'href',
+      'https://monitor.firefox.com/'
+    );
+  });
+
+  it('should show the lockwise icon and link', async () => {
+    const servicesList = MOCK_SERVICES.filter(
+      (item) => item.name === 'Firefox Lockwise'
+    );
+    renderWithRouter(
+      <MockedCache account={{ attachedClients: servicesList }}>
+        <ConnectedServices />
+      </MockedCache>
+    );
+    expect(await screen.findByTestId('lockwise-icon')).toBeTruthy;
+    expect(screen.getByTestId('service-name')).toHaveAttribute(
+      'href',
+      'https://www.mozilla.org/en-US/firefox/lockwise/'
+    );
+  });
+
+  it('should show the mobile icon and link', async () => {
+    const servicesList = MOCK_SERVICES.filter(
+      (item) => item.name === 'A-C Logins Sync Sample'
+    );
+    renderWithRouter(
+      <MockedCache account={{ attachedClients: servicesList }}>
+        <ConnectedServices />
+      </MockedCache>
+    );
+    expect(await screen.findByTestId('mobile-icon')).toBeTruthy;
+  });
+
+  it('should show the fpn icon and link', async () => {
+    const servicesList = MOCK_SERVICES.filter(
+      (item) => item.name === 'Firefox Private Network'
+    );
+    renderWithRouter(
+      <MockedCache account={{ attachedClients: servicesList }}>
+        <ConnectedServices />
+      </MockedCache>
+    );
+    expect(await screen.findByTestId('fpn-icon')).toBeTruthy;
+  });
+
+  it('should show the sync icon and link', async () => {
+    const servicesList = MOCK_SERVICES.filter(
+      (item) => item.name === 'Firefox Sync'
+    );
+    renderWithRouter(
+      <MockedCache account={{ attachedClients: servicesList }}>
+        <ConnectedServices />
+      </MockedCache>
+    );
+    expect(await screen.findByTestId('sync-icon')).toBeTruthy;
```

Comment:
Hmmm, not a blocker, but something to noodle on: I suspect we could turn the meat of each of these tests into a single function that takes a service name, and icon test ID, and instead use it for each test case. Could help dry things up. Again, not a blocker!

Reply:
Fixed this up